sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -26,6 +26,18 @@
 
 #include <azure/core/_az_cfg_prefix.h>
 
+/**
+ * @brief Azure IoT Hub Client method_twin_content_type option value: CBOR.
+ * @note It can be used by wrapping the macro in an #AZ_SPAN_FROM_STR macro as a parameter.
+ */
+#define AZ_IOT_HUB_CLIENT_OPTION_METHOD_TWIN_CONTENT_TYPE_CBOR "application%2Fcbor"
```

Comment:
Yup I would only suggest adding backticks to the method_twin_content_type so it renders in typewriter font in doxygen